fbpic/particles/deposition/cuda_methods.py

```diff
@@ -195,51 +195,43 @@ def deposit_rho_gpu_linear(x, y, z, w, q,
             # Mode 1
             R_m1_scal = wj * exptheta_m1
 
-            # Boundary Region Shifts
-            ir_lower = int64(math.floor(r_cell))
-
             R_m0_00 += r_shape_linear(r_cell, 0)*z_shape_linear(z_cell, 0) * R_m0_scal
             R_m0_01 += r_shape_linear(r_cell, 0)*z_shape_linear(z_cell, 1) * R_m0_scal
             R_m1_00 += r_shape_linear(r_cell, 0)*z_shape_linear(z_cell, 0) * R_m1_scal
             R_m1_01 += r_shape_linear(r_cell, 0)*z_shape_linear(z_cell, 1) * R_m1_scal
-
```

Comment:
This is the if condition that gets suppressed (I kept the `else:` part and unindented it). The "folding-in" of the charge deposited below the axis is done in the atomics operations further below (see calculation of the index `ir0`.)